components/builder-depot/src/backend/s3.rs

```diff
@@ -239,7 +239,7 @@ impl S3Handler {
                 let mut part_num: i64 = 0;
                 let mut should_break = false;
                 loop {
-                    let mut length = 0;
```

Comment:
I saw that compile error and couldn't figure out why it was complaining. I just chalked it up to the compiler being drunk again

Reply:
Hehe - yeah I thought some compiler bug initially as well..